pkg/fontconfig/build.zig

```diff
@@ -164,11 +164,23 @@ fn buildLib(b: *std.Build, module: *std.Build.Module, options: anytype) !*std.Bu
             "-DHAVE_SYS_STATVFS_H",
 
             "-DFC_CACHEDIR=\"/var/cache/fontconfig\"",
-            "-DFC_TEMPLATEDIR=\"/usr/share/fontconfig/conf.avail\"",
-            "-DFONTCONFIG_PATH=\"/etc/fonts\"",
-            "-DCONFIGDIR=\"/usr/local/fontconfig/conf.d\"",
             "-DFC_DEFAULT_FONTS=\"<dir>/usr/share/fonts</dir><dir>/usr/local/share/fonts</dir>\"",
         });
+
+        if (target.result.os.tag == .freebsd) {
+            try flags.appendSlice(&.{
+                "-DFC_TEMPLATEDIR=\"/usr/local/etc/fonts/conf.avail\"",
```

Comment:
The correct way to obtain the `/usr/local` part on FreeBSD is to query `sysctl user.localbase`. Hardcoding it would work for 99% cases, though.